package.json

```diff
@@ -20,14 +20,15 @@
     "aliasify": "1.9.0",
     "babel-cli": "6.26.0",
     "babel-eslint": "8.2.2",
-    "babel-jest": "22.4.1",
+    "babel-jest": "^22.4.3",
     "babel-plugin-react-transform": "3.0.0",
     "babel-plugin-transform-class-properties": "6.24.1",
     "babel-plugin-transform-object-rest-spread": "6.26.0",
     "babel-preset-es2015": "6.24.1",
     "babel-preset-react": "6.24.1",
     "babelify": "8.0.0",
     "browserify": "16.1.1",
+    "chokidar": "^2.0.3",
```

Comment:
what dependency do we have on chokidar? does it work if we just bump the version in the lock rather than defining it as a dependency?